src/algorithms/machinelearning/tensorflowpredictmusicnn.h

```diff
@@ -56,7 +56,7 @@ class TensorflowPredictMusiCNN : public AlgorithmComposite {
     declareParameter("graphFilename", "the name of the file containing the model to use", "", Parameter::STRING);
     declareParameter("input", "the name of the input nodes in the Tensorflow graph", "", "model/Placeholder");
     declareParameter("output", "the name of the node from which to retrieve the output tensors", "", "model/Sigmoid");
-    declareParameter("isTrainingName", "The name if the input reciving isTraining. If empty, it wont be feed ro the network", "", "");
+    declareParameter("isTrainingName", "the name of an additional input node to indicate the model if it is in training mode or not. Leave it empty when the model does not need such input", "", "");
```

Comment:
Still not sure, perhaps this will be an improvement? 

"the name of an additional input node indicating whether the model is to be run in a training mode (for models with a training mode, leave it empty otherwise)"